migrations/20221128133021_add_campaign_reports/migration.sql

```diff
@@ -0,0 +1,43 @@
+-- CreateEnum
+CREATE TYPE "campaign_report_file_type" AS ENUM ('photo', 'document');
+
+-- CreateTable
+CREATE TABLE "campaign_reports" (
```

Comment:
Creating new tables breaks our database backup job currently due to permission changes. Please whoever merges this PR, ping me in a private message on Discord when it's merged. I will need to adjust the permissions before this is shipped to prod

Reply:
Thanks for letting us know! Is there I can do on this PR to avoid breaking the backup job? Can you also please post a reference to this job, so that I can understand better what it does and why creating tables is breaking it :pray: